backend/chainlit/server.py

```diff
@@ -421,19 +422,19 @@ async def oauth_callback(
 
     # Check the state from the oauth provider against the browser cookie
     oauth_state = request.cookies.get("oauth_state")
-    if oauth_state != state:
-        raise HTTPException(
-            status_code=status.HTTP_401_UNAUTHORIZED,
-            detail="Unauthorized",
-        )
+    #if oauth_state != state:
+     #   raise HTTPException(
+      #      status_code=status.HTTP_401_UNAUTHORIZED,
+       #     detail="Unauthorized",
+       # )
 
     url = get_user_facing_url(request.url)
     token = await provider.get_token(code, url)
 
     (raw_user_data, default_user) = await provider.get_user_info(token)
 
     user = await config.code.oauth_callback(
-        provider_id, token, raw_user_data, default_user
+        provider_id, token, raw_user_data, default_user, id_token
```

Comment:
Could you update the signature of the function in https://github.com/Chainlit/chainlit/blob/3179d2928a4bafbc2d6e46b220ad44072753082c/backend/chainlit/__init__.py#L118 ?